src/Nox.Generator/NoxFileGenerator.cs

```diff
@@ -0,0 +1,171 @@
+﻿using Nox.Generator.Common;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using YamlDotNet.Core;
+using YamlDotNet.Serialization.NamingConventions;
+using YamlDotNet.Serialization;
+using Nox.Solution;
+using System.Reflection;
+
+namespace Nox.Generator;
+
+public class NoxFileGenerator
+{
+    private readonly List<string> _errors = new();
```

Comment:
@rochar do we need a debug file with listed errors (like Generator.cs) for NoxFileGenerator?